docs/source/sitemap.xml

```diff
@@ -0,0 +1,35 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  <url>
+    <loc>https://docs.kedro.org/en/stable/</loc>
+    <lastmod>2024-09-01T18:53:11.571849+00:00</lastmod>
```

Comment:
These dates have to be put here by hand, but we can try to guesstimate them so they are a bit more realistic @merelcht

Reply:
@lrcouto might be best if you update this and merge right before the release.